test/lib/adminUtilsTests.test.js

```diff
@@ -0,0 +1,333 @@
+import { expect } from 'chai';
+import fetchMock from 'fetch-mock';
+import sinon from 'sinon';
+
+import * as request from '../../src/lib/airtable/request';
+import * as userData from '../../src/lib/redux/userData';
+import { store } from '../../src/lib/redux/store';
+
+import {
+  toggleValidColor,
+  validateField,
+  removeOwner,
+  getOwnerRecordsForProjectGroup,
+  inviteMember,
+  triggerEmail
+} from '../../src/lib/adminUtils';
+
+describe('adminUtils', () => {
+  const sandbox = sinon.createSandbox();
+
+  describe('#toggleValidColor', () => {
+    it('should return "b-is-not-valid" for a nonempty input with type 0', () => {
+      const result = toggleValidColor('testcolor', 0);
+
+      expect(result).to.be.a('string');
+      expect(result).to.equal('b-is-not-valid');
+    });
+    it('should return "b-is-valid" for a blank input with type 0', () => {
+      const result = toggleValidColor('', 0);
+
+      expect(result).to.be.a('string');
+      expect(result).to.equal('b-is-valid');
+    });
+    it('should return "b-is-valid" for an undefined input with type 0', () => {
+      const result = toggleValidColor(undefined, 0);
+
+      expect(result).to.be.a('string');
+      expect(result).to.equal('b-is-valid');
+    });
+    it('should return "\u00a0" for a false input with type 1', () => {
```

Comment:
@jassol `/u00a0` is a unicode character code for nonbreaking space (just had to look that up lol). In general any code that starts with `/u` can be assumed to be a unicode character.